### PR TITLE
{Network} Azure Bastion ssh/rdp calls bug fixes

### PR DESCRIPTION
**Pull request by aavalang**
**Description**

Fixing issue with get token call which was being called to a method that is supposed to be deprecated and move to get_raw_token.

Also fixing an issue with logging where it returns types of auth-types.

**Testing Guide**

Tested locally on machine to check if feature works as expected.

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Network

**Review by kairu-ms — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/network/tunnel.py [hunk 1]
```diff
@@ -74,14 +74,13 @@ def is_port_open(self):
...
-        user = profile.get_current_account_user()
         # Generate an Azure token with the VSTS resource app id
-        auth_token = profile.get_access_token_for_resource(user, None, None)
+        auth_token, _, _ = profile.get_raw_token()
         content = {
             'resourceId': self.remote_host,
             'protocol': 'tcptunnel',
             'workloadHostPort': self.remote_port,
-            'aztoken': auth_token,
+            'aztoken': auth_token[1],
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/tunnel.py:83]: @jiasli Is it the correct way to get auth token?

Comment (jiasli) [reply src/azure-cli/azure/cli/command_modules/network/tunnel.py:83]: Yes

**Review by jiasli — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/network/tunnel.py [hunk 2]
```diff
@@ -74,14 +74,13 @@ def is_port_open(self):
 
     def _get_auth_token(self):
         profile = Profile(cli_ctx=self.cli_ctx)
-        user = profile.get_current_account_user()
         # Generate an Azure token with the VSTS resource app id
-        auth_token = profile.get_access_token_for_resource(user, None, None)
+        auth_token, _, _ = profile.get_raw_token()
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/network/tunnel.py:78]: By the way, the default `resource` for the access token is ARM. Will it work?

**Pull request merged**